app/controllers/api/v1/activities_controller.rb

```diff
@@ -1,14 +1,20 @@
 class Api::V1::ActivitiesController < Api::BaseController
-  respond_to :json, :yaml, :on => [:latest, :just_updated]
 
   def latest
     @rubygems = Rubygem.latest(50)
-    respond_with(@rubygems, :yamlish => true)
+    render_rubygems
   end
 
   def just_updated
     @rubygems = Version.just_updated(50).map(&:rubygem)
-    respond_with(@rubygems, :yamlish => true)
+    render_rubygems
   end
 
+  private
+  def render_rubygems
+    respond_to do |format|
+      format.json { render json: @rubygems }
+      format.yaml { render yaml: @rubygems, yamlish: true }
```

Comment:
@arthurnn Hi Arthur, is the `yamlish` option necessary? Because I saw you use this somewhere in this commit, but not for all of the `render yaml:...` :-)